frontend/packages/console-dynamic-plugin-sdk/src/extensions/catalog.ts

```diff
@@ -17,6 +17,8 @@ export type CatalogItemType = ExtensionDeclaration<
     filters?: CatalogItemAttribute[];
     /** Custom groupings specific to the catalog item. */
     groupings?: CatalogItemAttribute[];
+    /** Indicated the sub-catalog is enabled or not */
+    enabledType?: boolean;
```

Comment:
Is this really needed? Isn't it possible to just skip these entries?

Reply:
I will remove `enabledType`